crates/api/src/linker.rs

```diff
@@ -395,7 +395,26 @@ impl Linker {
         Instance::new(module, &imports)
     }
 
-    fn import_get(&self, import: &ImportType) -> Option<&Extern> {
+    /// Returns the [`Store`] that this linker is connected to.
+    pub fn store(&self) -> &Store {
+        &self.store
+    }
+
+    /// Returns an iterator over all items defined in this `Linker`.
```

Comment:
Could you add a brief mention of what the components of the tuple return are? Looking at the code, I'm not surprised it's module, name, extern, but it feels like something worth making explicit in the docs.